lib/registries/addon/components/registration-form-view/styles.scss

```diff
@@ -19,3 +19,105 @@
 .Answer > p {
     font-size: 16px;
 }
+
+.NavigationMenu {
+    position: absolute;
```

Comment:
Could this use `<ResponsiveDropdown>` instead of reimplementing how a dropdown works?